webapp/client/public/css/base.css

```diff
@@ -312,7 +316,6 @@ ol li::before {
 
 .optional-hint {
   font-size: var(--text-medium);
```

Comment:
font-weight removed?

Reply:
It is now in the font-face def and therefore not needed here. In most cases we use the regular BundesSansWeb which has a weight of 400 (defined in body/html)